lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -64,19 +64,22 @@
 
 module ActiveRecord
   module ConnectionHandling # :nodoc:
-    # Establishes a connection to the database that's used by all Active Record objects.
-    def oracle_enhanced_connection(config) # :nodoc:
+    def oracle_enhanced_connection_class # :nodoc:
```

Comment:
Looks like we're probably going to rename this to `<adapter_name>_adapter_class`, I'll make that change ASAP.